### PR TITLE
jonpan resubmitting lab05 on a new branch

### PR DESCRIPTION
**Pull request by notvert**
Moved winning ticket outside of loop as requested by Bruce

### CODE REVIEW

**Comment by brucestull** [comment]
Original Pull Request (which was closed so student could create new branch without merge conflicts): https://github.com/PdxCodeGuild/class_HB2/pull/63

**Review by brucestull — CHANGES_REQUESTED** [review]
[Pick6](https://github.com/PdxCodeGuild/class_HB2/blob/main/1%20Python/labs/05%20Pick6.md)

**Needs Work**

Comments:
* Lab needs Version 2, display the ROI: "The ROI (return on investment) is defined as (earnings - expenses)/expenses. Calculate your ROI, print it out along with your earnings and expenses."

File: code/jonpan/lab05.py [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+        balance += 100
+    elif match_counter == 4:
+        balance += 50000
+    elif match_counter == 5:
+        balance += 1000000
+    elif match_counter == 6:
+        balance += 25000000
+
+payoff = balance + ticket_cost
+ROI = (payoff - ticket_cost) / payoff
```
Comment (brucestull) [inline code/jonpan/lab05.py:36]: * ROI should be calculated using formula in lab document. Make sure to use the variables so the ROI calcualtion uses the specific form below:
```
roi = (earnings - expenses)/expenses
```

**Review by brucestull — DISMISSED** [review]
[Pick6](https://github.com/PdxCodeGuild/class_HB2/blob/main/1%20Python/labs/05%20Pick6.md)

**Needs Work**

Comments:
* Lab should use the provided equation for ROI:
```
roi = (earnings - expenses)/expenses
```

File: code/jonpan/lab05.py (hunk 1 above)
Comment (notvert) [reply code/jonpan/lab05.py]: Changes submitted

**Review by nathanBonilla — APPROVED** [review]
Great job Jon!

**Pull request merged**